app/assets/stylesheets/metrics/_code-climate-deparments.scss

```diff
@@ -27,3 +27,30 @@ table.code-climate-projects-report {
     color: black;
   }
 }
+
+.code-climate-summary-container {
+  background-color: gainsboro;
+}
+
+.code-climate-summary-title {
+  padding: 0.5em 4em;
+}
+
+.code-climate-summary-info {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 4em;
+}
+
+.code-climate-summary-ratings {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.code-climate-summary-issues {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
```

Comment:
maybe in the future but since bootstrap has some utils we could use them, i can not remember exactly how but it is something like `class='d-flex justify-content'`

Reply:
Yeah, I've seen that used in the code I replaced. Do we prefer using those classes instead of creating ours and setting the styles in the css files?
It's just my opinion (and not a strong one, for sure, so I'd love to hear your thoughts) that creating our own classes may add semantics to the code, and that using those classes does not sound very different than adding the styles inline in the html (like `style='display: flex; justify-content: space-between;'`)

Reply:
I like to have new css classes with more sense than the ones that bootstrap already has, but maybe we can reduce the amount, wdyt? For example .code-climate-summary-ratings and .code-climate-summary-issues are the same, could we use something more general in this case to wrap both?

Reply:
Another way to wrap all these classes could be:
```
.code-climate-summary {
  # general styles...
  .container { ... }
  .title { ... }
  .info { ... }
}
```

Reply:
I like that alternative! And thinking about both classes you mentioned, I could join them in something like `code-climate-summary-item-list`, since that's basically what they have in common.
Thanks!